lib/runner/job.ex

```diff
@@ -36,24 +36,17 @@ defmodule ElixirBench.Runner.Job do
   @doc """
   Executes a benchmarking job for a specific commit.
   """
-  def start_job(id, repo_slug, branch, commit, config) do
+  def start_job(job, opts) do
     ensure_no_other_jobs!()
 
-    job = %Job{
-      id: to_string(id),
-      repo_slug: repo_slug,
-      branch: branch,
-      commit: commit,
-      config: config
-    }
```

Comment:
I see you moved the responsibility. I think that's a good thing, that the job module just deals with job structs here so I like it :+1:

Reply:
Yeah.. regarding the field names, I created an issue #11 to make them compatible later